robot/Cumulus/resources/locators_47.py

```diff
@@ -16,6 +16,7 @@
     },
     "record": {
         'button': "//div[@class='actionsContainer']/button[@title='{}']",
+        'footer':"//div[@class='footer active' or contains(@class,'footer-visible')]",
```

Comment:
is it really "footer<space>active" but "footer<dash>visible? It's a shame they aren't consistent "footer active" and "footer visible" or "footer-active" and "footer-visible".

Reply:
unfortunately yes there is no consistency :(